src/IssueViz.Security/OpenInIDE/Http/ShowHotspotOwinRequestHandler.cs

```diff
@@ -53,6 +53,7 @@ internal ShowHotspotOwinRequestHandler(IOpenInIDERequestHandler openInIDERequest
 
         public async Task ProcessRequest(IOwinContext context)
         {
+            context.Response.Headers["Access-Control-Allow-Origin"] = context.Request.Headers["Origin"];
```

Comment:
It might be better to do it as part of OwinPipelineProcessor.ProcessRequest, instead of in each handler.